src/playground-project.ts

```diff
@@ -479,7 +481,9 @@ export class PlaygroundProject extends LitElement {
           // immediately. We'll get back here again after it updates via a
           // CONNECT_PROJECT_TO_SW message from the proxy.
           console.info(
-            'Playground service worker is outdated, waiting for update.'
+            `Playground service worker is outdated. ` +
+              `Want ${version} but got ${e.data.version}. ` +
+              `Waiting for update.`
```

Comment:
is there a corresponding message after this that the update happened?

Reply:
To show in the console? No, but the playground will start working so it's pretty obvious.